extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -31,13 +31,17 @@ require([
 		//Fallback to the generic playlist when no recommended videos playlist is set for the wiki
 		recommendedPlaylist = videoDetails.recommendedVideoPlaylist || 'Y2RWCKuS',
 		inAutoplayCountries = geo.isProperGeo(instantGlobals.wgArticleVideoAutoplayCountries),
-		willAutoplay = featuredVideoCookieService.isAutoplayEnabled() && inAutoplayCountries,
+		willAutoplay = isAutoplayEnabled() && inAutoplayCountries,
```

Comment:
What if the user will not have the cookie set but will be located in one of the autoplay countries?

Reply:
I know it's an old code, just asking out of curiosity :)

Reply:
It will autoplay then